velox/docs/functions/spark/conversion.rst

```diff
@@ -2,6 +2,8 @@
 Conversion Functions
 ====================
 
+Casting from UNKNOWN type to all other scalar types are supported, e.g., cast(NULL as int).
```

Comment:
typo: "Casting.. are supported" -> "Casting... is supported"

Reply:
@mbasmanova, just fixed it and rebased the code. Thanks!